app/core/log.py

```diff
@@ -1,13 +1,45 @@
 import logging
 import logging.config
 import queue
+from enum import Enum
 from logging.handlers import QueueHandler, QueueListener
 from pathlib import Path
 from typing import Any
 
+import uvicorn
+
 from app.core.config import Settings
 
 
+class ModifiedFormatter(uvicorn.logging.DefaultFormatter):
```

Comment:
Maybe `ColoredConsoleFormatter` ?
```suggestion
class ColoredConsoleFormatter(uvicorn.logging.DefaultFormatter):
```